core/arch/arm/mm/tee_mmu.c

```diff
@@ -263,7 +263,7 @@ TEE_Result tee_mmu_init(struct tee_ta_ctx *ctx)
 TEE_Result tee_mmu_map(struct tee_ta_ctx *ctx, struct tee_ta_param *param)
 {
 	TEE_Result res = TEE_SUCCESS;
-	paddr_t pa;
+	paddr_t pa = NULL;
```

Comment:
OK, this refactoring is probably best left for another PR ;-) In any case I'm fine with the initialization to NULL.

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`